utilities/cli/holohub.py

```diff
@@ -210,6 +210,11 @@ def _create_parser(self) -> argparse.ArgumentParser:
             dest="with_operators",
             help="Optional operators that should be built, separated by semicolons (;)",
         )
+        run.add_argument(
+            "--docker_opts",
```

Comment:
@tbirdso, thanks so much for the update. I'm just wondering if there is any reason we are combining underscore with dashes. I think it makes more sense to have `--docker-opts` instead of `--docker_opts`. IMO, it would be easier to remember and type arguments which only have dashes (no underscores). Is there any benefit of mixing them?

Reply:
Thanks for the update.